rest/rest_blackbox_test.go

```diff
@@ -108,6 +108,57 @@ func TestReplaceDomainPrefixInAbsoluteURLOK(t *testing.T) {
 	assert.Equal(t, "https://core.openshift.io/testpath6", urlStr)
 }
 
+func TestReplaceDomainPrefixInAbsoluteURLStr(t *testing.T) {
+	tables := []struct {
+		urlStr    string
+		replaceBy string
+		relative  string
+		want      string
+		wantErr   bool
+	}{
+		{
+			urlStr:    "http://auth.localhost",
```

Comment:
We are not really interested in testing dummy test/default URLs. Could we add tests for "http**s**://auth.openshift.io" and "http**s**://auth.prod-preview-openshift.io" URLs?